gulp-tasks/test.js

```diff
@@ -367,13 +367,23 @@ function getFiles() {
     return wfHelper.promisedExec(cmd, '.')
     .then(function(results) {
       let files = [];
+      let warnForSideEffect = false;
       results.split('\n').forEach(function(filename) {
-        if (RE_SRC_BASE.test(filename) || RE_DATA_BASE.test(filename) ||
-            RE_GULP_BASE.test(filename) || filename === 'app.yaml' ||
-            filename == 'gulpfile.js') {
-              files.push(filename);
+        if (RE_GULP_BASE.test(filename) || filename === 'gulpfile.js') {
+          warnForSideEffect = true;
+          files.push(filename);
+        } else if (RE_SRC_BASE.test(filename) || RE_DATA_BASE.test(filename) ||
+                   filename === 'app.yaml') {
+          files.push(filename);
         }
       });
+      if (warnForSideEffect === true) {
+        const warn = chalk.yellow('WARNING:');
```

Comment:
I won't like, this repo is so liberal with warnings I doubt anyone will pay attention to this.